src/main.cpp

```diff
@@ -125,6 +125,7 @@ void usage()
   std::cerr << "    BPFTRACE_PERF_RB_PAGES      [default: 64] pages per CPU to allocate for ring buffer" << std::endl;
   std::cerr << "    BPFTRACE_NO_USER_SYMBOLS    [default: 0] disable user symbol resolution" << std::endl;
   std::cerr << "    BPFTRACE_CACHE_USER_SYMBOLS [default: auto] enable user symbol cache" << std::endl;
+  std::cerr << "    BPFTRACE_VERIFY_LLVM_IR     [default: 0] verify generated LLVM IR" << std::endl;
```

Comment:
Not exactly a user facing env var IMO. Maybe document in test docs somewhere instead?

Reply:
Good point, I removed it from the help.